components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -2291,6 +2292,12 @@ public void makeAPIKeysForwardCompatible(API api) throws APIManagementException
             apiMgtDAO.makeKeysForwardCompatible(new ApiTypeWrapper(api), oldVersion
             );
         }
+        int apiId = apiMgtDAO.getAPIID(api.getId(), null);
+        APIEvent apiEvent = new APIEvent(UUID.randomUUID().toString(), System.currentTimeMillis(),
```

Comment:
don't we hv to send subscription related events?

Reply:
resolved